doc/reference/vm_live_migration_internals.md

```diff
@@ -0,0 +1,127 @@
+---
+discourse: ubuntu:[Online&#32;VM&#32;live-migration&#32;(QEMU&#32;to&#32;QEMU)](50734)
+---
+
+(vm-live-migration-internals)=
+# LXD VM live migration implementation
+
+LXD supports {ref}`live-migration-vms` of virtual machines by streaming instance state from a source {abbr}`QEMU (Quick Emulator)` to a target QEMU. VM live migration is supported for all storage pool types.
+
+API extension: `migration_vm_live`
+
+## Conceptual process
+
+The live migration workflow varies depending on the type of storage pool used. The two key scenarios are non-shared storage and shared storage within a cluster (e.g., Ceph). If live state transfer is not supported by a target, a stateful stop is performed prior to migration.
+
+### Live migration for non-shared storage
+
+This process leverages the QEMU built-in Network Block Device (NBD) client-server mechanism to transfer the virtual machine's disk and memory state. Below is an overview of the steps:
+
+1. Set up connection.
+1. Determine migration type (shared or non-shared storage).
+1. Set migration capabilities.
+1. Non-shared storage preparation.
+- Create and configure snapshot file for root disk writes during migration;
```

Comment:
Due to the bullet points that are used in the middle, right now the RTD build shows this list like this:
<img src="https://github.com/user-attachments/assets/97c77bb9-baae-4091-a834-e85bef2b737f" width="100px" />

Where I think you want the last numbers to be 5 through 7. As a workaround, you could use 1.s for the first four numbers, then 5 for the fifth, and either 6.s for both the last two, or 6. and 7. That would work, but I think ultimately it would be more confusing to mix auto-numbering and manual numbering like that. I advise reverting to manual numbering in this case, as the simplest and most easy-to-maintain solution.

Reply:
Agreed - I think manual numbering is better in this case.